drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -15,7 +15,7 @@ if ETH_SAM_GMAC
 config ETH_SAM_GMAC_QUEUES
 	int "Number of active hardware TX and RX queues"
 	default 1
-	range 1 $(dt_node_int_prop_int,/soc/ethernet@40050088,num-queues)
+	range 1 6
```

Comment:
I think it would be better to just remove `range` here since this can be confusing/misleading, especially when using menuconfig.

The value is going to be checked in the driver code anyway.